build/generate_syncset.py

```diff
@@ -1,31 +1,37 @@
 #!/usr/bin/env python
 
-import oyaml as yaml
-
 import os
-import sys
 import argparse
 import copy
 
+import oyaml as yaml
+
+
+TPL = "00-osd-managed-cluster-validating-webhooks.selectorsyncset.yaml.tmpl"
```

Comment:
TPL sounds like a TLA, could we not shorten unless it's a very common shortening?  Recommend using `TEMPLATE` at a minimum.